src/Breadcrumb/EzContentBreadcrumbBuilder.php

```diff
@@ -0,0 +1,39 @@
+<?php
```

Comment:
Lets update the filename to `src/Breadcrumb/EzcontentBreadcrumbBuilder.php`. Notice small letter `c`